modules/manage/pages/kubernetes/storage/volume-types.adoc

```diff
@@ -0,0 +1,208 @@
+
+- *Storage location*: Can be backed by various storage systems including local storage on the worker node or cloud providers, such as AWS EBS, Azure Disk, and Google Persistent Disk.
+
+- *Provisioning*: PVs can be provisioned either statically, where the admin creates a PV, or dynamically, where the system automatically creates a PV in response to a PVC.
+
+- *Suitability for the Redpanda data directory*: Recommended for safeguarding Redpanda topics and crucial data.
+
+- *Suitability for the Tiered Storage cache*: Strikes a balance between persistence and performance. Ensures cache data is preserved, reducing fetches from primary storage.
+
+TIP: Make regular backups of PersistentVolumes to reduce the risk of data loss in cases where the volume is lost.
+
+=== Local and remote storage
+
+PersistentVolumes can be backed by local or remote storage:
+
+[cols"1a,2a,1a"]
+|===
+| PersistentVolume | Description| Best Use
+
+| Local (recommended)
+| Local PersistentVolumes store Redpanda data on a local disk that's attached to the worker node. Redpanda reads and writes data faster on a local disk.
+However, local disks are tied to the lifecycle of the worker node.
+If the worker node fails for any reason, you may lose access to the Redpanda data.
+| High throughput and low latency
+
+| Remote
+| Remote storage persists data outside of the worker node so that the data is not tied to a specific worker node and it can be recovered.
+| High availability
+|===
+
+=== Reclaim policies
+
+PersistentVolumeClaims are not deleted when Redpanda brokers are removed from a cluster. It is your responsibility to delete PersistentVolumeClaims when they are no longer needed. As a result, it's important to consider your volume's reclaim policy:
+
+* *Delete*: The associated storage is removed once the PVC is deleted.
+* *Retain*: Even after the PVC is deleted, the actual storage persists. You can reclaim it with a new PVC.
+
+TIP: Considering the importance of data,  Redpanda Data recommends the `Retain` policy to ensure no accidental data loss.
+
+=== Persistent Volume Provisioners in Kubernetes
+
+Kubernetes offers two methods to provision Persistent Volumes (PVs): static provisioning and dynamic provisioning.
+
+==== Static provisioners
+
+Static provisioners require manual intervention from administrators. An administrator pre-provisions a set of PersistentVolumes. These volumes are physical storage spaces within the cluster that are ready for use but aren't bound to any specific PersistentVolumeClaims (PVCs) yet. When a PVC is created, the Kubernetes control plane looks for a PV that matches the requirements of the PVC and binds them together.
+
+When you use a static provisioner, You must create one PersistentVolume for each Redpanda broker. When the Redpanda Helm chart is deployed, an existing PersistentVolume in the cluster is selected and bound to one PersistentVolumeClaim for each Redpanda broker.
```

Comment:
When using a static provisioner, you must create one PersistentVolume for each Redpanda broker. When the Redpanda Helm chart is deployed, an existing PersistentVolume in the cluster is selected and bound to one PersistentVolumeClaim for each Redpanda broker.